test/e2e/scheduling/predicates.go

```diff
@@ -851,3 +854,15 @@ func creatHostPortPodOnNode(f *framework.Framework, podName, ns, hostIP string,
 		framework.ExpectNoError(err)
 	}
 }
+
+func createRandomPort() int32 {
```

Comment:
This will run on the machine executing the tests correct? I think there is an assumption here that the tests are executing on the same machine as an all-in-one cluster under test (or at least the only schedulable node in the cluster).

I'm wondering if we can't just choose a fixed uncommon high port if these run serially.